applications/GeoMechanicsApplication/custom_utilities/math_utilities.cpp

```diff
@@ -0,0 +1,24 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//  Main authors:    Anne van de Graaf
+//
+
+#include "math_utilities.hpp"
+
+
+namespace Kratos
+{
+
+double DegreesToRadians(double AngleInDegrees)
```

Comment:
```suggestion
double DegreesToRadians(const double AngleInDegrees)
```

Reply:
I'm not sure whether this actually improves the code. I have checked the C++ Core Guidelines and there they explicitly mention this situation as an exception to the rule that objects should be immutable whenever possible. From [section Con 1: By default, make objects immutable](https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#con1-by-default-make-objects-immutable) it says (under the heading Exception):

> Function parameters passed by value are rarely mutated, but also rarely declared `const`. To avoid confusion and lots of false positives, don’t enforce this rule for function parameters.

Also the accompanying example illustrates this:
```
void g(const int i) { ... }  // pedantic
```

Since I don't see a good reason to deviate from that recommendation here, I'd like to keep it the way it is. (Unless you have a convincing reason why we should make it immutable here.) Thanks.